packages/adelus/src/Adelus_defines.h

```diff
@@ -74,7 +74,7 @@
 #elif defined(DREAL)
 
   #define ADELUS_DATA_TYPE double
-  #undef COMPLEX
+  #undef ADELUS_COMPLEX
```

Comment:
Should use Kokkos Complex, but that is fine for this PR.

Reply:
Thanks, @srajama1 !